header.php

```diff
@@ -19,8 +19,9 @@
 // Send the Content-type header in case the web server is setup to send something else
 header('Content-type: text/html; charset=utf-8');
 
-// Prevent site from being embedded in a frame
-header('X-Frame-Options: deny');
+// Prevent site from being embedded in a frame, if that's not what we want
+if (!defined('FORUM_EMBEDDED_IFRAME'))
+	header('X-Frame-Options: deny');
```

Comment:
Sorry for criticizing all of these...

What I had in mind was something along these lines:
~~~php
$frame_options = defined('FORUM_FRAME_OPTIONS') ? FORUM_FRAME_OPTIONS : 'deny';
header('X-Frame-Options: '.$frame_options');
~~~

Thanks for taking care of all these! =)